### PR TITLE
Rename time

### PR DESCRIPTION
**Pull request by GuillaumeDSM**
requires https://github.com/Drakkar-Software/OctoBot-Backtesting/pull/178

### CODE REVIEW

File: octobot_trading/personal_data/portfolios/portfolio_manager.py [hunk 1]
```diff
@@ -84,7 +85,7 @@ async def handle_balance_update_from_order(self, order, require_exchange_update:
         portfolio changes using order data (as in trading simulator)
         :return: True if the portfolio was updated
         """
-        if self.trader.is_enabled:
+        if self.trader.is_enabled and self._enable_portfolio_update_from_order:
```
Comment (Herklos) [inline octobot_trading/personal_data/portfolios/portfolio_manager.py:88]: Could we add a test for it please? To be sure it won't be broken with future changes

**Review by Herklos — COMMENTED** [review]
:+1:

File: octobot_trading/personal_data/portfolios/portfolio_manager.py (hunk 1 above)
Comment (GuillaumeDSM) [reply octobot_trading/personal_data/portfolios/portfolio_manager.py:88]: sure, i will

Comment (GuillaumeDSM) [reply octobot_trading/personal_data/portfolios/portfolio_manager.py:88]: done

File: tests/personal_data/portfolios/test_portfolio_manager.py [hunk 2]
```diff
@@ -75,6 +72,13 @@ async def test_handle_balance_update_from_order(backtesting_trader):
         _refresh_real_trader_portfolio_mock.assert_called_once()
         _refresh_simulated_trader_portfolio_from_order_mock.assert_not_called()
         _refresh_real_trader_portfolio_mock.reset_mock()
+        with portfolio_manager.disabled_portfolio_update_from_order():
+            await portfolio_manager.handle_balance_update_from_order(order, False)
+            _refresh_real_trader_portfolio_mock.assert_not_called()
+            _refresh_simulated_trader_portfolio_from_order_mock.assert_not_called()
+            await portfolio_manager.handle_balance_update_from_order(order, True)
+            _refresh_real_trader_portfolio_mock.assert_not_called()
+            _refresh_simulated_trader_portfolio_from_order_mock.assert_not_called()
```
Comment (Herklos) [inline tests/personal_data/portfolios/test_portfolio_manager.py:81]: :+1:

**Review by Herklos — APPROVED** [review]

**Review by Herklos — APPROVED** [review]

**Pull request merged**